docs/core/migration/20-21.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Migrate from NET Core 2.0 to 2.1
+description: Learn how to upgrade your .NET Core 2.0 app to 2.1.
+author: mairaw
+ms.author: mairaw
+ms.date: 06/15/2018
+---
+# Migrate from NET Core 2.0 to 2.1
+
+This article shows you how the basic steps for you to migrate your .NET Core 2.0 app to 2.1. If you're looking to migrate your ASP.NET Core application to 2.1, see [Migrate from ASP.NET Core 2.0 to 2.1](/aspnet/core/migration/20_21).
```

Comment:
nit: for you to migrate --> for migrating
nit: use of "app" in the first sentence, but "application" in the second; they should be consistent